packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -206,6 +206,7 @@ const MembersView = () => {
               orgMembers={orgMembersNotInThisTeam}
               members={team.members}
               teamId={team.id}
+              justEmailInvites={true}
```

Comment:
Hey, there's no need to add any new props. If we find a use case to re-add the username in the future, we can manage that at that time. For now, just remove the conditional rendering for the label and only add 'enter_email'

Reply:
@anikdhabal Ok! One question - `justEmailInvites` prop is already present in the `MemberInvitationModal` component. So, should I just remove that prop also from there, along with the conditional renderings?

Reply:
Yeah, pls remove that.

Reply:
@anikdhabal Done!

Also FYI - Updated the video of the PR description.

Reply:
Great